AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -460,7 +460,7 @@ lavocedimanduria.it,gioialive.it,terlizzilive.it#%#//scriptlet('prevent-setTimeo
 schwaebische.de##.goodies
 community-z.com#@#.ad-placement
 myfxbook.com#%#//scriptlet('set-cookie', 'lastAdBlockerMessageDisplayedDate', '1')
-client.googiehost.com##.banner
+htmlgames.com,client.googiehost.com##.banner
```

Comment:
I think it's better to add this domain to the same rule in english specific as an ad leftover

Reply:
Why so? It is clearly an adb message.